zoho_mcp/tools/api.py

```
@@ -332,47 +508,89 @@ async def zoho_api_request_async(
+                            if response.status_code == 204:  # No Content
+                                result = {
+                                    "status": "success",
+                                    "message": "Operation completed successfully"
+                                }
+                                # Cache successful GET responses
+                                if cache_key:
+                                    _set_cached_response(cache_key, result)
+                                return result
+                            return {"text": response.text}
+                            
+                except httpx.HTTPStatusError as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)